content/client-server-api/modules/receipts.md

```diff
@@ -87,6 +122,84 @@ not have their notification counts rewound to that point in time. While
 uncommon, it is considered valid to have an `m.read` (public) receipt lag
 several messages behind the `m.read.private` receipt, for example.
 
+##### Threaded read receipts
+
+{{% added-in v="1.4" %}}
+
+If a client does not use [threading](#threading), then they will simply only
+send "unthreaded" read receipts which affect the whole room regardless of threads.
+
+Threading introduces a concept of multiple conversations being held in the same
+room and thus deserve their own read receipts and notification counts. An event
+is considered to be "in a thread" if it has either a `rel_type` of `m.thread` or
+has child events with a `rel_type` of `m.thread` (in which case it'd be the
+thread root). Events not in a thread but still in the room are considered to be
+part of the "main timeline", or a special thread with an ID of `main`.
+
+A threaded read receipt is simply one which has a `thread_id` on it, targeting
```

Comment:
I would probably move that paragraph 1 level up so that it follows the reference of "unthreaded". As it is mentioned, but not defined.
This paragraph addresses that